README.md

```diff
@@ -28,42 +28,35 @@ the code and integrating with the Pulumi ecosystem.
 ### Provisioning a virtual machine on Google Cloud Platform
 
 ```kotlin
-import com.pulumi.googlenative.compute.v1.kotlin.instance
+import com.pulumi.gcp.compute.kotlin.ComputeFunctions
+import com.pulumi.gcp.compute.kotlin.instance
 import com.pulumi.kotlin.Pulumi
 
 fun main() {
     Pulumi.run { ctx ->
-        val instance = instance("google-native-sample-project") {
+        val debianImage = ComputeFunctions.getImage {
+            family("debian-11")
+            project("debian-cloud")
+        }
+    
+        val instance = instance("gcp-sample-project") {
```

Comment:
(minor)

I know this was named like that before this PR, but maybe "gcp-sample-instance" would be a better name here?